lib/scarpe/glibui/document_root.rb

```diff
@@ -8,13 +8,12 @@ class GlimmerLibUIDocumentRoot < Scarpe::GlimmerLibUIWidget
 
     def initialize(properties)
       @callbacks = {}
-
       super
     end
 
-    def display
+    def display(properties = {})
       <<~GTEXT
-        window {
+        window("#{properties["title"] || "Scarpe with GlimmerLibUI"}", #{properties["width"] || 640}, #{properties["height"] || 480}) {
```

Comment:
would you believe that even 90% of the work of this line was done by copilot

I think it's a testament that as we write multiple display services and mindfully structure our code that copilot is actually going to be able to help us out, as it sees how we put things together